Python/bytecodes.c

```diff
@@ -3226,40 +3326,6 @@ dummy_func(
             _SAVE_RETURN_OFFSET +
             _PUSH_FRAME;
 
-        inst(CALL_PY_WITH_DEFAULTS, (unused/1, func_version/2, callable, self_or_null, args[oparg] -- unused)) {
```

Comment:
Just curious, this isn't worth specializing anymore? I forget exactly why it can't be converted to tier two (I'm guessing that there's more than one reason, since the use of `this_instr` can be easily worked around).

Reply:
Two reasons:
1. Because this loops over both arguments and defaults, it probably isn't much faster than `CALL_PY_GENERAL` for tier 1. It is almost certainly slower in the JIT because of its size. `CALL_PY_GENERAL` calls a help function, so is more compact.
2. We expect that, for 3.14, the tier 2 optimizer will convert them both to the same optimal sequence of operations.